actions/debootstrap_action.go

```diff
@@ -3,6 +3,8 @@ Debootstrap Action
 
 Construct the target rootfs with debootstrap tool.
 
+NB: file "/etc/resolv.conf" will be removed after execution.
```

Comment:
This probably need a bit more explanation as reading it without knowing the details this seems odd